On aarch64 platforms setting filters fails when using memset on the DPM:

[ 2950.913528] Unable to handle kernel paging request at virtual address ffff000013f30604
[ 2950.921509] pgd = ffff800063c3f000
[ 2950.924936] [ffff000013f30604] *pgd=0000000000000000
[ 2951.002398] PC is at __memset+0x4c/0x1d0
[ 2951.006345] LR is at OS_Memset+0x14/0x20 [netanalyzer]

[ 2951.468495] Call trace:
[ 2951.555677] [<ffff0000084245cc>] __memset+0x4c/0x1d0
[ 2951.560667] [<ffff00000111cc8c>] netana_set_filter+0x94/0x1b8 [netanalyzer]
[ 2951.567645] [<ffff00000111cf30>] netana_open_device+0x180/0x218 [netanalyzer]
[ 2951.574797] [<ffff0000011168e4>] netana_open+0xdc/0x148 [netanalyzer]
[ 2951.581240] [<ffff0000081f6d8c>] chrdev_open+0xa4/0x1a0
[ 2951.586466] [<ffff0000081ee9d4>] do_dentry_open+0x164/0x320
[ 2951.592038] [<ffff0000081efe34>] vfs_open+0x44/0x70
[ 2951.596917] [<ffff000008201410>] path_openat+0x318/0x12e8
[ 2951.602316] [<ffff000008203608>] do_filp_open+0x80/0xf8
[ 2951.607541] [<ffff0000081f0288>] do_sys_open+0x170/0x218
[ 2951.612852] [<ffff0000081f0368>] SyS_openat+0x10/0x18
[ 2951.617905] [<ffff0000080839dc>] __sys_trace_return+0x0/0x4
[ 2951.623478] Code: d65f03c0 cb0803e4 f2400c84 54000080 (a9001d07)
[ 2951.629663] ---[ end trace 9cb851bcc65a6860 ]---

diff -aurN a/OS_Linux.c b/OS_Linux.c
--- a/OS_Linux.c	2019-08-12 11:06:39.614505008 +0200
+++ b/OS_Linux.c	2019-08-27 14:47:03.175200745 +0200
@@ -145,7 +145,9 @@
 /*****************************************************************************/
 void OS_Memset(void* pvMem, unsigned char bFill, uint32_t ulSize)
 {
-        memset(pvMem, bFill, ulSize);
+    uint8_t *dst = (uint8_t*)pvMem;
+    while(ulSize-- > 0)
+        *dst++ = bFill;
 }
 
 /*****************************************************************************/
